reinforcement_learning/README.md

```diff
@@ -6,6 +6,8 @@ These examples demonstrate how to train reinforcement learning models on SageMak
 
 **IMPORTANT for rllib users:** Some examples may break with latest [rllib](https://docs.ray.io/en/latest/rllib.html) due to breaking API changes. Please refer to [Amazon SageMaker RL Container](https://github.com/aws/sagemaker-rl-container) for the latest public images and modify the configs in entrypoint scripts according to [rllib algorithm config](https://docs.ray.io/en/latest/rllib-algorithms.html).
 
+If you are using PyTorch rather than TensorFlow, please set `debugger_hook_config=False` when calling `RLEstimator()` to avoid TensorBoard conflicts.
```

Comment:
What is the conflicts here? And isn't `debugger_hook_config` by default is `False`?

Reply:
If not specified, a default one is created using the estimator's ``output_path``, unless the region does not support SageMaker Debugger ([link](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/estimator.py#L212)). `tf.summary` generated by ray cannot be picked up appropriately by SM Debgger.